.travis.yml

```diff
@@ -20,5 +21,5 @@ before_install:
 
 script:
     - python setup.py test -a "--verbose"
-    - find . -type f -name "*.py" ! -name 'test_*.py' | xargs flake8 --max-line-length=100
+    - find . -type f -name "*.py" ! -name 'test_*.py' ! -name 'demo_*.py' | xargs flake8 --max-line-length=100
```

Comment:
Skipping the demos in the syntax check because they are kind of messy. It would be nice to get clean notebooks instead.